helper_thread.c

```diff
@@ -126,7 +126,7 @@ static void submit_action(enum action a)
 
 	ret = write_to_pipe(helper_data->pipe[1], &data, sizeof(data));
 	if (ret != 1) {
-		fprintf(stderr, "failed to write action into pipe, err %i:%s", errno, strerror(errno));
+		log_err("failed to write action into pipe, err %i:%s", errno, strerror(errno));
```

Comment:
Didn't you just add this one in patch 1?

Reply:
Clarify, please, I didn't get

Reply:
You add this fprintf() in the first patch, and then a later patch changes it. Just use log_err() in the first patch, and you then have one less place to fix up?

Reply:
Either that, or do the fprintf -> log_err change first. Then it's already consistent by the time you add your addition.

Reply:
@axboe got you. done